model/src/exec_env.rs

```diff
@@ -180,4 +181,8 @@ impl<T: Env> EnvHandler<T> {
     pub fn block_number(&self) -> T::BlockNumber {
         T::block_number()
     }
+
+    pub fn dispatch_call(&self, call: T::Call) {
+        T::dispatch_raw_call(call.encode().as_slice())
```

Comment:
```suggestion
        T::dispatch_raw_call(call.into().encode().as_slice())
```

Reply:
Also please adjust all uses of `env.dispatch_call` by removing the superflous `.into()` in its params.

Reply:
Done in https://github.com/paritytech/ink-types-node-runtime/pull/3